models/Swatch/Swatch.js

```diff
@@ -0,0 +1,36 @@
+const uuid = require('uuid-v4');
+const Color = require('../Color/Color');
+
+/**
+ * Also known as 'Color Variables'
+ */
+class Swatch {
+  static get Model() {
+    return {
+      _class: 'swatch',
+    };
+  }
+
+  /**
+   *
+   * @param {Object} args
+   * @param {String} args.name
+   * @param {Color} args.color
+   * @param {Color.Model} json
+   */
+  constructor(args, json) {
+    if (json) {
+      Object.assign(this, Swatch.Model, json);
+      this.value = new Color(this.value);
+    } else {
+      const id = args.id || uuid().toUpperCase();
+      Object.assign(this, Swatch.Model, {
+        name: args.name || 'Name',
```

Comment:
Sketch defaults to the color's hex value as the name, should we do the same thing here?

Reply:
Sorry - missed the notification for this over the holidays - I'll take a look at your feedback over the weekend